scripts/build.py

```diff
@@ -712,6 +725,14 @@ def minify_func(source_path, target_path, file_hashes, filename):
     elif filename.endswith('.css') or filename.endswith('.js'):
         print 'Minifying %s' % source_path
         _minify(source_path, target_path)
+    elif filename.endswith('.ts'):
+        compile_typescript_files(source_path)
+        target_path = target_path.replace('.ts', '.js')
```

Comment:
This is more of an intermediate_path, and the next variable would be the target_path.

Reply:
Updated.